src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs

```diff
@@ -112,6 +189,12 @@ public static CheckResult CheckPermission(string privilege)
         /// <since_tizen> 4 </since_tizen>
         public static void RequestPermission(string privilege)
         {
+            if (!s_PrivilegesInProgress.Add(privilege))
+            {
+                Log.Error(LogTag, "Request for this privilege: " + privilege + " is already in progress.");
+                throw new ArgumentException("Request for this privilege: " + privilege + " is already in progress.");
+            }
+
```

Comment:
I added it in 290 line

Reply:
I have to throw Exception because delegate `RequestResponseCallback` will not be called.

When `ppm_request_permissions` returns error it just end execution and `RequestMultipleResponseCallback` delegate will not be called.